### DISCUSSION TITLE
"create torrent" still not possible?

### DISCUSSION THREAD

**Discussion by iqigravity** [jesec/flood#139] [Q&A]
Hi,
I followed your thread #50  and tried to create a new torrent from an existing folder. Before I installed flood from scratch and after running for the irst time I clicked on the + -button and I'm still missing the "create"-tab you mentioned in #50 . 

What did I do wrong? Thanks in advance!
iqi

**Comment by jesec** [comment]
I can not reproduce the issue.

My bet is that you haven't install the latest version of Flood. Follow the README, install Flood from correct repository and make sure that the right version is started. Flood shows its version in Settings -> About:

![image](https://user-images.githubusercontent.com/11585141/103187839-ff7ad600-4900-11eb-8945-571ff901cf4f.png)

Additionally, clean the cache of your browser.

**Comment by iqigravity** [comment]
You were right. In my how-to it said

git clone https://github.com/jfurrow/flood.git
instead of
git clone https://github.com/jesec/flood.git

But the same software from two authors? Who has really invented it?

iqi

**Reply by jesec** [reply]
John created the original Flood project. I am maintaining it right now.

**Reply by iqigravity** [reply]
Sad that they aren't "compatible".  Thought I just clone your repo and then copy config.js and databases into the installation and it will do it.
But it didnt work out.

**Reply by jesec** [reply]
There have been several major revisions. It is now much easier to install, configure and run Flood. 

Technically, you can just copy databases to new runtime directory (`~/.local/share/flood`) and convert the config changes to CLI arguments. Though, it is just easier to start from scratch.
